PacketDefinitions420/PacketHandlerManager.cs

```diff
@@ -258,17 +259,32 @@ public bool BroadcastPacket(byte[] data, Channel channelNo, PacketFlags flag = P
 
         public bool HandlePacket(Peer peer, byte[] data, Channel channelId)
         {
-            var header = new PacketHeader(data);
-            var convertor = GetConvertor(header.Cmd, channelId);
+            var reader = new BinaryReader(new MemoryStream(data));
+            RequestConvertor convertor;
 
-            switch (header.Cmd)
+            if (channelId == Channel.CHL_COMMUNICATION || channelId == Channel.CHL_LOADING_SCREEN)
```

Comment:
I think you should also add the `ChannelID.QuickChat` channel to here. Not sure if it is used here, but should be generally.

Reply:
I'm opening it as an issue. Ignore this.